docs/src/main/paradox/cassandra.md

```diff
@@ -1,6 +1,6 @@
 # Cassandra Connector
 
-The Cassandra connector provides a way to provide the result of a Cassandra query as a stream of rows.
+The Cassandra connector allows you to read and write to Cassandra. You can query a stream of rows from @scaladoc[CassandraSource](akka.stream.alpakka.cassandra.scaladsl.CassandraSource$) or use prepared statements to insert or update with @scaladoc[CassandraSink](akka.stream.alpakka.cassandra.scaladsl.CassandraSink$).
```

Comment:
I dont see any of the other connectors doing that. It renders as a inline link. Im not sure about how to style it. It would be weird to interrupt every sentence with java-link & scala-link.

Do you have any suggestion? I'd rather leave it.

Reply:
It looks weird only in the markdown source, not in the produced page, right?

I would be cool to have a `@unidoc` directive that figures out how to link to both Scala and Java docs automatically. Preferably even checking that those classes indeed exist? Way outside the scope of this PR though :).